WalletWasabi.Fluent/Views/CoinSelection/SelectCoinsDialogView.axaml

```diff
@@ -0,0 +1,145 @@
+        </StackPanel>
+
+      </Grid>
+
+    </controls:ContentArea.Caption>
+
+    <controls:ContentArea.Styles>
+      <Style Selector="TreeDataGridRow">
+        <Setter Property="Focusable" Value="False" />
+        <Setter Property="Height" Value="47.5" />
+      </Style>
+
+      <Style Selector="TreeDataGridRow ContentPresenter">
+        <Setter Property="VerticalAlignment" Value="Center" />
+      </Style>
+
+      <Style Selector="TreeDataGridTemplateCell > ContentPresenter">
+        <Setter Property="Margin" Value="5 0 10 0"></Setter>
+      </Style>
+
+      <Style Selector="ContentPresenter > TextBlock">
+        <Setter Property="FontFamily" Value="{StaticResource MonospacedFont}" />
+      </Style>
+
+      <Style Selector="TextBox.FilterBox">
+        <Setter Property="Transitions">
+          <Setter.Value>
+            <Transitions>
+              <DoubleTransition Property="Opacity" Duration="0:0:0.25" />
+            </Transitions>
+          </Setter.Value>
+        </Setter>
+      </Style>
+
+      <Style Selector="TextBox.FilterBox.Visible">
+        <Setter Property="Opacity" Value="1" />
+      </Style>
+      <Style Selector="TextBox.FilterBox.Hidden">
+        <Setter Property="Opacity" Value="0" />
+      </Style>
+
+    </controls:ContentArea.Styles>
+
+    <DockPanel>
+      <DockPanel DockPanel.Dock="Top" Margin="8" LastChildFill="False">
+        <DockPanel.Styles>
+          <Style Selector="TextBox /template/ DataValidationErrors">
+            <Setter Property="IsVisible" Value="False" />
+          </Style>
+        </DockPanel.Styles>
+        <CheckBox x:Name="GroupByLabels" DockPanel.Dock="Left" Content="Group by Labels" />
+        <TextBox Classes="FilterBox"
+                 Width="200"
+                 Classes.Visible="{Binding #GroupByLabels.IsChecked}"
+                 Classes.Hidden="{Binding !#GroupByLabels.IsChecked}"
+          Text="{Binding LabelBasedSelection.Filter, Mode=TwoWay}"
+          Watermark="Filter" Margin="20 0 0 0">
+        </TextBox>
+      </DockPanel>
+
+      <StackPanel DockPanel.Dock="Bottom">
+
+        <StackPanel DockPanel.Dock="Bottom" x:Name="Summary" Margin="0 8">
+          <TextBlock>
+            <TextBlock.Text>
+              <MultiBinding StringFormat="{}{0} selected in {1} coin/s">
+                <Binding Path="SelectedAmount^" Converter="{x:Static converters:MoneyConverters.ToFormattedStringBtc}" />
+                <Binding Path="SelectedCount^" />
+              </MultiBinding>
+            </TextBlock.Text>
+          </TextBlock>
+
+          <ProgressBar Maximum="{Binding TargetAmount.Satoshi}" Value="{Binding SelectedAmount^.Satoshi}" Margin="0 4 0 0" />
+
+          <TextBlock TextAlignment="Right" Classes="h9 monoSpaced" Opacity="0.6"
+                     Text="{Binding RemainingAmount^, StringFormat='{}{0} remaining', Converter={x:Static converters:MoneyConverters.ToFormattedStringBtc}}">
```

Comment:
If https://github.com/zkSNACKs/WalletWasabi/pull/8975/files#r961350782 is removed then:
 ```suggestion
                     Text="{Binding RemainingAmount^, StringFormat='{}{0} BTC remaining', Converter={x:Static converters:MoneyConverters.ToFormattedStringBtc}}">
```